library/ajax/pharmacy_autocomplete/index.php

```diff
@@ -0,0 +1,42 @@
+<?php 
+require_once('../../../interface/globals.php');
+
+
+?>
+
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+  <link rel="stylesheet" href="../../js/jquery-ui.min.css" type="text/css" /> 
```

Comment:
remove lines 12, 22, 23 and then place following code here:
``````
<?php
html_header_show();
$include_standard_style_js = array("jquery-ui","jquery-ui-sunny");
require "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
?>